superchic.sh

```diff
@@ -1,6 +1,7 @@
 package: SuperCHIC
 version: "%(tag_basename)s%(defaults_upper)s"
-tag: "v2.04"
+tag: "alice/v2.04"
+source: https://github.com/hcab14/SuperCHIC.git
```

Comment:
Better if we had it under alisw (ALICE official). If it's on a user repo (albeit known and trustable as in this case) nothing prevents him/her from taking it out or change it without prior warning.